### ISSUE TITLE
"Open database object" / F4 it's not working for partitioned tables in PostgreSQL

### ISSUE THREAD

**Issue by ionutursuleanu** [dbeaver/dbeaver#12652]
#### System information: 
- macOS Big Sur 11.4
- DBeaver 21.0.5

#### Connection specification:
- PostgreSQL 11.9

#### Describe the problem you're observing:
"Open database object" / F4 it's not working for partitioned tables. It does nothing
#### Steps to reproduce, if exist:
Run
```sql
CREATE TABLE measurement (
    city_id         int not null,
    logdate         date not null,
    peaktemp        int,
    unitsales       int
) PARTITION BY RANGE (logdate);

CREATE TABLE measurement_y2006m02 PARTITION OF measurement
    FOR VALUES FROM ('2006-02-01') TO ('2006-03-01');

CREATE TABLE measurement_y2006m03 PARTITION OF measurement
    FOR VALUES FROM ('2006-03-01') TO ('2006-04-01');
```
Reconnect and try to open `measurement` table or any of its partitions.

**Comment by kseniiaguzeeva** [comment]
I cannot reproduce it in DB Navigator and SQL Editor. 

https://user-images.githubusercontent.com/51405061/120074154-2aca4280-c0a4-11eb-8494-0dfa19e80ffc.mov

Can you open other objects? Open an object by double-click/via context menu?
Please also try Fn+F4.
If nothing helps, please send [logs](https://github.com/dbeaver/dbeaver/wiki/Log-files) and a video.

**Comment by ionutursuleanu** [comment]
Hi @kseniiaguzeeva ,

From Database Navigator it works for me also. See attached video when it's not working. This is happening only for partitioned tables.


https://user-images.githubusercontent.com/6136404/120075645-daa2ae80-c0aa-11eb-8d24-7dc02b72afc2.mov

**Comment by kseniiaguzeeva** [comment]
I could reproduce it. Thank you for the report. As a workaround please add schema_name before a table_name.

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.2.1

**Comment by Matvey16** [comment]
Verified
